src/components/Table/Table.stories.tsx

```diff
@@ -167,3 +168,17 @@ export const RowState: Story = {
     rowState: (record: TableRecordState) => (record.state?.toLowerCase() as RowStateEnum),
   },
 }
+
+export const Empty = () : ReactElement => {
```

Comment:
I'd use a better name, since the table is not empty. Leaving a suggestion based on the prop.

```suggestion
export const UnpaginatedScrollableTable = () : ReactElement => {
```

but of course, consider a better one if you have it in mind.

Reply:
Totally right, I've renamed the story as FitParentHeight, the name of the prop used